lib/aruba/processes/basic_process.rb

```diff
@@ -115,7 +115,13 @@ def before_run; end
       def after_run; end
 
       def inspect
-        out = stdout(:wait_for_io => 0) + stderr(:wait_for_io => 0)
+        out = stdout(:wait_for_io => 0)
```

Comment:
Oh... Can you give me an example. This sounds weird... I would like to understand what can cause this code to fail...